docs/markdown/end-of-life.md

```diff
@@ -1,34 +1,34 @@
 # End of life statements
-The currently supported release train of PowerDNS is 3.x. The development train is 4.x.  
-2.x is end of life.
+The currently supported release train of PowerDNS, for both the Authoritative Server and the Recursor is 4.x.
 
-## PowerDNS Authoritative Server 2.x
-21st of May 2015
+PowerDNS Authoritative Server 3.4 is considered legacy will only receive critical bug fixes and security fixes.
+
+PowerDNS Authoritative Server 3.3 will only receive security fixes.
+
+PowerDNS Recursor 3.7 is considered legacywill only receive critical bug fixes and security fixes.
```

Comment:
this sentence is at least missing a space between "legacy" and "will", and perhaps a **and** as suggested above